gemfiles/rails_8.1.gemfile

```diff
@@ -10,14 +10,14 @@ group :development do
   gem "csv"
   gem "nokogiri"
   gem "pry-byebug"
-  gem "rails", "~> 6.1.0"
+  gem "rails", ">= 8.1.0.rc1", "< 8.2"
   gem "rspec"
   gem "sequel"
-  gem "sqlite3", "~> 1.7.0"
+  gem "sqlite3", "~> 2.1.0"
 
   group :mongo do
     gem "bson"
-    gem "mongoid", "~> 9.0"
+    gem "mongoid", github: "mongodb/mongoid"
```

Comment:
Why was this change required?

Reply:
It looks like it all started here: https://github.com/talkable/datagrid/commit/487e1debf30dfe50e7b70599f49bcd9272016b4a#diff-635b20966b1d2fea76ee0565254eab939d55f6d22e3c4727cf9d453f589b87a0R24

I am not sure why this was done. Maybe at a time mongoid didn't work with Rails 7. Can you check if you can use a stable version of mongoid here?